packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -261,6 +264,8 @@ class GoogleMap extends StatefulWidget {
   /// * The My Location button animates to focus on the user's current location
   /// if the user's location is currently known.
   ///
+  /// This feature is not present in the Google Maps SDK for the web.
```

Comment:
Ideally we wouldn't reference a specific platform here, but the ship has sailed on that below so we can do this now, and revisit the whole thing in the future.